src/content/blog/new-domain-new-discord-same-project.md

```diff
@@ -0,0 +1,60 @@
+grateful for. However, we're trying to improve our experience as much as possible for both new users and contributors to the project. At this point 
+we feel we can move away from Astrolicious, the community organization who's been helping us nurture our little project, and go in alone to have full 
+control over our own ecosystem! After talking to [Florian](https://github.com/florian-lefebvre) and [Alex](https://github.com/alexanderniebuhr), 
+we've come to the conclusion that it's time for a fresh coat of new paint. Therefore, I introduce to you:
+
+## The new StudioCMS Discord Server
+That's right, we've escaped the confines of our single channel on Astrolicious' Server and made ourselves [a new home](https://chat.studiocms.dev). This 
+will be the place for you to chat and receive news about StudioCMS from now on! *Don't worry, the blog isn't going anywhere ;)*
+
+Moving to the new Discord Server has already allowed us to do some pretty cool things. For example, yours truly has been hard at work at our new Open 
+Source Discord Bot, [Apollo](https://github.com/withstudiocms/apollo)! If you're on the Astro Discord, you'll know what they're good at. PTAL notifications, 
+moderation tools (soon™️), support forum utilities, it's all there.
+
+You can even use this Bot in your own Servers! All you need is a GitHub App, Discord App, and a libSQL database connection, for which we obviously use 
+and highly recommend [Turso](https://turso.tech/). You can take a look at the [README file](https://github.com/withstudiocms/apollo?tab=readme-ov-file#apollo) 
+to learn more.
+
+But if you really **did** read the title (or clicked one of the links in this article), you might've also noticed another thing. Thus, it is with great 
+pleasure that I announce... *Drumroll please...* 🥁
+
+## The new StudioCMS GitHub Org
+That's right, along with the Discord Server we've also got [our own GitHub Organization](https://github.com/withstudiocms) now! 
+Similar to the Discord, we've felt that only having access to the repository without any organization-management options was holding us back a tiny bit, 
+so here's to [github.com/withstudiocms](https://github.com/withstudiocms).
+
+This new Org allows us to have better team management, de-clutter our Monorepo and iterate faster on any changes we want to implement. It's small things, 
+really - for example, our website now has its own repository. It's not much, but it's definitely better than having to navigate an entire monorepo to update a blog post. 
+*Speaking from experience here.*
+
+Okay, we've got one last change. Take a look at your browser's address bar. Do you see that? Yeah, that's right. **I introduce to you:**
+
+## studiocms.dev
+That's right, I just realized I said "That's right" below every heading. Terribly sorry for that.
+
+In other news, we've moved to a new domain! `studiocms.dev` is going to be our new home starting from now on. According to ChatGPT, it's supposed to give us a 
```

Comment:
```suggestion
To wrap things up, we've moved to a new domain! [studiocms.dev](https://studiocms.dev/) will be our new home from now on. According to ChatGPT, it's supposed to give us a 
```

Reply:
Unsure on this one, I like the joke about me accidentally using the same two words below every heading. Willing to do
```md
That's right, I just realized I said "That's right" below every heading. Terribly sorry for that.

To wrap things up, we've moved to a new domain! [studiocms.dev](https://studiocms.dev/) will be our new home from now on. According to ChatGPT, it's supposed to give us a ...
```

Reply:
Nope, all good if you want to keep it! Need to undo part of one of my other suggestions first though